src/test/java/org/jboss/forge/arquillian/util/DependencyUtilTest.java

```diff
@@ -77,7 +77,7 @@ public void should_get_all_available_versions_if_not_supported_by_chameleon() th
         deps.add(DependencyBuilder.create().setVersion("7.1").getCoordinate());
 
         List<String> dep = DependencyUtil.toVersionString(deps,
-            createContainer("arquillian-jbossas-managed-4.2", "JBoss As"));
+            createContainer("org.jboss.as", "arquillian-jbossas-managed-4.2", "JBoss As"));
 
         assertThat(dep).contains("4.1", "7.1");
```

Comment:
@bartoszmajsak there is no weakness in `DependencyUtil.toVersionString`, as we are getting all available co-ordinates using ` dependencyFacet.resolveAvailableVersions`, So before calling `DependencyUtil.toVersionString` we will get list of only realistic dependencies using `dependencyFacet.resolveAvailableVersions`.

We are creating test data instead of using `dependeFacet` here, for this test. 

I have renamed variables & modified test to make it more readable.